docs/reference/models.rst

```diff
@@ -0,0 +1,23 @@
+Models
+======
+
+.. currentmodule:: graspy.models
+
+Erdos-Reyni models
+------------------
+
+.. autoclass:: EREstimator
+
+.. autoclass:: DCEREstimator
+
+Stochastic block models
+-----------------------
+
+.. autoclass:: SBEstimator
```

Comment:
because if we are going to do that then we would have RDPGMEstimator, ERMEstimator... they are all models in the `models` module so I don't see the point in adding M to each of them. These are all estimators for a model. @jovo @j1c opine?

Reply:
just as sklearn does not call their thingy GaussianMixtureModel() https://scikit-learn.org/stable/modules/generated/sklearn.mixture.GaussianMixture.html#sklearn.mixture.GaussianMixture.bic

Reply:
If I am understanding this correctly, @hhelm10 's point is that there is an explicit `M` in `SBM`, while there isn't necessarily the word `Model` in `RDPG` or `ER`? 

Also calling it `SBMEstimator` makes it a bit more consistent with simulations since we call the simulation function `sbm` not `sb`.